go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/k0sproject/k0sctl
 
 go 1.15
 
+replace github.com/segmentio/analytics-go v3.1.0+incompatible => github.com/kke/analytics-go v1.2.1-0.20210209122110-10364370169e
```

Comment:
No clue how that version number was generated. The commit hash is 10364370169e934c834178453b2b8004aa72744f.